server/mediaserver_test.go

```diff
@@ -48,6 +67,123 @@ func (s *StubSegmenter) SegmentRTMPToHLS(ctx context.Context, rs stream.RTMPVide
 	return nil
 }
 
+func TestStartBroadcast(t *testing.T) {
+	s := setupServer()
+
+	// Empty discovery
+	mid := core.ManifestID(core.RandomVideoID())
+	storage := drivers.NodeStorage.NewSession(string(mid))
+	pl := core.NewBasicPlaylistManager(mid, storage)
+	if _, err := s.startBroadcast(pl); err != ErrDiscovery {
+		glog.Error("Expected error with discovery")
```

Comment:
Why are we doing `glog.Error` instead of `t.Errof` in `TestStartBroadcast`?

Reply:
Typo! Good catch, fixed.

Note that we use `glog.Error` in a few other places such as `SegmentRTMPToHLS` which is right above this one.